integration-tests/localconfig-example.yaml

```diff
@@ -11,3 +11,8 @@ services:
           name: general
           webhook_url: some-webhook-url
           channel_id: some-channel-id
+  aws:
```

Comment:
I wonder how consistent should we be in this file. Shouldn't we use `aws-tnb` instead of `aws`? asking because we have `slack-tnb` few lines above.

Reply:
> 

@jomarko - yes, I do agree with you. But this file is not on us, this is config file just used by TNB framework, you can have a look at how the config file look like [here](https://github.com/tnb-software/TNB/tree/main/system-x) - It's expecting that name `aws` and not `aws-tnb`

Hope it answers your question?
Thanks for your feedback!

Reply:
The config file item name is derived from credentials if specified in [account instance](https://github.com/tnb-software/TNB/blob/f2a8abb84faeecda4d96dd9638b1d0b2050c8599/system-x/services/aws/aws-common/src/main/java/software/tnb/aws/common/account/AWSAccount.java#L11).